src/pudl/transform/ferc1.py

```diff
@@ -130,7 +135,14 @@ class WideToTidyXbrl(TransformParams):
     """
 
 
-def wide_to_tidy_xbrl(df: pd.DataFrame, params: WideToTidyXbrl) -> pd.DataFrame:
+class WideToTidy(TransformParams):
+    """Parameters for converting either or both XBRL and DBF table from wide to tidy."""
+
+    xbrl = WideToTidyDataSource()
+    dbf = WideToTidyDataSource()
```

Comment:
Should the `xbrl` and `dbf` attributes have types explicitly associated with them as well as default values? 
```py
    xbrl: WideToTidyDataSource = WideToTidyDataSource()
    dbf: WideToTidyDataSource = WideToTidyDataSource()
```

Reply:
ah good catch!